app/models/project.rb

```diff
@@ -163,6 +163,20 @@ def remove_keyword! name, user
                    project: self)
   end
 
+  def similar_projects
+    if self.keywords.empty?
```

Comment:
Better this way, ``` return [] if keywords.empty? ```. We should only use `self.attribute` for assignments :smile_cat: